modules/service_accounts/outputs.tf

```diff
@@ -1,4 +1,9 @@
 output "aws_iam_instance_profile" {
   value = aws_iam_instance_profile.tfe.name
+
+  description = <<-EOD
+  The name of the IAM instance profile to be attached to the TFE EC2 instance(s) which is authorized to access the S3 
+  storage buckets and EC2 autoscaling group.
+  EOD
```

Comment:
Nice love the line-limit commenting on this one.

Reply:
Can't let a little limit get in the way of verbosity!